package/gluon-mesh-vpn-fastd/check_site.lua

```diff
@@ -1,4 +1,4 @@
-local fastd_methods = {'salsa2012+gmac', 'salsa2012+umac', 'null+salsa2012+gmac', 'null+salsa2012+umac', 'null'}
+local fastd_methods = {'salsa2012+umac', 'null+salsa2012+umac', 'null@l2tp', 'null'}
```

Comment:
Without looking at the fastd release notes - were these methods removed or why are they invalid now?

Reply:
I decided to remove them from the default configuration of the OpenWrt fastd package, so they are not available anymore (as it was recommended not to use them since the introduction of the UMAC methods).